packages/datadog-plugin-gcloud-pubsub/src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+function createWrapRequest (tracer, config) {
+  return function wrapRequest (request) {
+    return function wrappedRequest (cfg = { reqOpts: {} }, cb) {
+      let topic
+      if (cfg.reqOpts) {
+        topic = cfg.reqOpts[cfg.method === 'createTopic' ? 'name' : 'topic']
+      }
+      const tags = {
+        component: 'google-cloud-pubsub',
+        'resource.name': topic || 'default',
```

Comment:
Also, since the method is not in the span name, should it go in the resource name in front of the topic?

Reply:
> Is the default topic literally called default?

No. There is not default topic, and you're required to make one. If there's no topic then something is very wrong so we should probably just pass the falsy value through.

> Also, what is the cardinality of topics in general?

I don't think this is too huge per application unless people are misusing topics. We're doing the same thing in `rhea`, FWIW.

Reply:
> Also, since the method is not in the span name, should it go in the resource name in front of the topic?

That's not too bad. How about `${method}::${topic}`? Note that a topic actually has the form `project/${projecctId}/topic/${topicName}`.

Reply:
> If there's no topic then something is very wrong so we should probably just pass the falsy value through.

Yep, definitely do that for all occurrences of this.

> How about ${method}::${topic}?

Usually a space is used for other integrations, so let's stick with that to be consistent.